lib/event-source.js

```diff
@@ -0,0 +1,79 @@
+var mongojs = require('mongojs');
+var inflect = require('i')();
+var _ = require('lodash');
+var EventSource = function() {
+};
+
+EventSource.prototype.init = function(router, adapter, options) {
+  this.router = router;
+  this.adapter = adapter;
+  this.options = options;
+  this.db = mongojs((process.env.OPLOG_MONGODB_URL || process.argv[3]) + '?slaveOk=true');
+}
+
+EventSource.prototype.initRoute = function(routeName) {
+  this.router.get(routeName + '/changes', this.handler.bind(this));
+};
+
+EventSource.prototype.handler = function(req, res) {
+    var me = this;
+    var routeName = /\/(.+)\/changes/.exec(req.route.path)[1];
+    var query = {
+      ns : this.options.db + '.' + routeName,
+    };
+    
+    if (req.query.event) query.op = req.query.event;
+    if (req.query.seq) {
+      var operation = req.query.seq.split('=');
+      //stop this block if we're being sent bad ops
+      if (_.contains(['gt', 'lt', 'gte', 'lte'], operation[0])) { 
```

Comment:
using lt, lte implies a bounded collection
can't see how this adds value to a change events stream, could you elaborate a bit ?

Not against range filtering, however it's not part of the spec : 
https://github.com/agco/json-api-search-profile/blob/master/public/change-events-profile.md 

Perhaps we should have a chat first before we build this in ?